config.json

```diff
@@ -266,6 +266,16 @@
         "Maybe"
       ]
     },
+    {
+      "uuid": "d1dc946d-a142-42ed-abec-e53605635c0d",
+      "slug": "twelve-days",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
+      "topics": [
+      ]
+    },
+    
```

Comment:
I'll remove this extra line (otherwise I hear rumours that there is an automatic JSON formatter in the works that would remove it, so might as well get ahead of the curve)